otter/cloud_client.py

```diff
@@ -345,13 +347,14 @@ def _log_it(result):
         # So we can link it to any non-cloud_client logs
         request_id = resp.request.headers.getRawHeaders(
             'x-otter-request-id', [None])[0]
-
+        resp_body = (
+            json.dumps(response_body_filter(json_body), sort_keys=True)
```

Comment:
do you know why we wanted json in the first place? why not just always log it structured?